src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,345 @@
+
+The following example of a narrative slide may make this difference clearer:
+
+.. code-block:: markdown
+
+  # [Which dataset is this?](https://nextstrain.org/community/inrb-drc/ebola-nord-kivu)
+
+  The dataset displayed here differs if you run it on localhost (auspice)
+  or through nextstrain.org.
+
+  When viewed on nextstrain.org this dataset is sourced from
+  [this GitHub repository](https://github.com/inrb-drc/ebola-nord-kivu)
+  as it is using our
+  [community sharing functionality](https://docs.nextstrain.org/en/latest/guides/share/community-builds.html).
+
+  To work when running on auspice locally, we would need to have a dataset
+  with the following filename: `community_inrb-drc_ebola-nord-kivu.json`.
+
+
+
+This complexity encouraged us to build the interactive debugger we have used in this tutorial and we encourage to use that approach rather than developing your narratives using Auspice locally & then trying to share them through nextstrain.org!
+
+
+Where can I find some example narratives to use as inspiration?
+---------------------------------------------------------------
+
+All of the narratives written by the core nextstrain team are available in `this GitHub repo <github.com/nextstrain/narratives>`__.
+
+Future directions
+-----------------
+
+The app used to test narratives is in its infancy.
+We hope to incrementally add features such as markdown editing, changing the dataset view settings, and the ability to publish the narrative to nextstrain groups.
+The eventual aim is to be able to write & publish an entire narrative from within the app, without needing to know any specifics of the markdown language behind it.
+
+
+Bugs? Improvements? Suggestions?
+================================
+
+The debugger (as of 2022-01-08) is in a beta-release phase. Please get in touch if you have suggestions or find bugs!
+You can submit an `issue on GitHub <https://github.com/nextstrain/nextstrain.org/>`__ or make a post on our `discussion forum <discussion.nextstrain.org/>`__.
+
+
+
+
+--------------
+
+.. rubric:: Footnotes
+
+.. [#f1] The provided string is actually rendered as ``### <abstract>``, so the first line will appear as a h3 heading!
+  Multiple line strings are possible in YAML and we suggest using these.
+
+.. [#f2] In practice, the URL path component is what is used to access the dataset relative to the current hostname, however for historical reasons the protocol and hostname must be either ``http://localhost:4000`` or ``https://nextstrain.org``. 
+
+.. [#f3] This isn't great from a file-size point of view, and the markdown file isn't nice to look at itself, but it may be the best current solution!
```

Comment:
I think a nice user experience would be to have `nextstrain remote upload` (and any other narrative uploading/editing clients) auto-embed images this way. It makes the narrative self-contained, which is really nice. (Think back to the bad old days when most Word docs didn't embed images in them by default, but merely referenced them on the filesystem. It was bad.)

Reply:
I, happily, do not remember those dark days 😂 

Would be a nice user experience for locally referenced images for sure - the alternatives get pretty complicated pretty fast

Reply:
Yeah, locally available images would be auto-embedded but hotlinked/remote images could be left alone. With a web-based editor, dragging and dropping an image would directly embed (or upload to somewhere we control and then hotlink, but I think this has lots of downsides).